tools/config/go-templates/api.mustache

```diff
@@ -166,48 +166,6 @@ func (a *{{{classname}}}Service) {{#lambda.camelcase}}{{{nickname}}}{{/lambda.ca
 		return {{#returnType}}localVarReturnValue, {{/returnType}}nil, reportError("{{paramName}} is required and must be specified")
 	}
 	{{/isPathParam}}
-	{{#minItems}}
```

Comment:
would it be useful to have this info as doc or we'll be encouraging teams to remove this info from OpenAPI completely?

Reply:
No. min, max, and pattern are very valid information that can be used in the documentation and other form.
We need to make sure that this info is correct but also do not need to impose validation for it in SDK